float64 velocity_desired
uint16 pwm_max 
uint16 pwm_neutral
uint32 KP
uint32 KD
uint16 rosrate
